Rover/sailboat.h

```diff
@@ -75,6 +75,12 @@ class Sailboat
         USE_MOTOR_ALWAYS = 2
     };
 
+    // states of SAIL_CONTROL_TYPE parameter and sail_control_type variable
+    enum SailControlType {
```

Comment:
should be an enum class to so it doesn't collide with global namespace for variable like "FIXED"

Reply:
thanks for the suggestion tridge. this is now fixed by defining SailControlType as enum class and casting *sail_control_type* to int